.github/composite_actions/log_cw_metric_wrapper/action.yaml

```diff
@@ -0,0 +1,103 @@
+name: Log CW Metric Wrapper
```

Comment:
Wrapper for log_cw_metric

Auto fills: 
repo: ${{ github.repository }}
run-id: ${{ github.run_id }}

And calls configure AWS credentials

So that workflows that need to log metrics don't need to worry about these extra details.